test/service/test_response.py

```diff
@@ -2613,76 +2625,117 @@ def tearDownClass(cls):
         cls._teardown_indices()
         super().tearDownClass()
 
-    def _count_total_cells(self, response_json):
+    @attr.s(auto_attribs=True, frozen=True)
+    class CellCounts:
+        project_cells: int
+        cells_by_organ: Dict[str, int]
```

Comment:
They type hint doesn't reflect the optionality. PyCHarm shows me a warning about this.